index.html

```diff
@@ -13,6 +13,263 @@
+                    <li><input type="Email" id="email" name="email" placeholder="scdk@yahoo.com" required></li>
+                  </ul>
+                  <textarea name="user_message" id="msg" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
+                  <button class="ctn-btn">See Project</button>
+                </form>
+            </div>
+            <footer>
+              <hr>
+              <div class="btm-icon">
+                <i class="fab fa-github fa-2x"></i>
+                <i class="fab fa-facebook fa-2x"></i>
+                <i class="fab fa-instagram fa-2x"></i>
+                <i class="fab fa-twitter fa-2x"></i>
+              </div>
+              <p class="text-align">emailhere@email.com</p>
+            </footer>
+    </div>
+    <div class="sub-grid-6">
+      <h2 class="text-align">CONTACT US</h2>
+      <p class="text-align">If you have an application you are<br> interested in developing, a feature that you<br> need
+        built or a project that needs coding.<br> I’d love to help with it</p>
+        <div class="tp-form">
+          <form class="text-align" action="https://formspree.io/f/form_id" method="post">
+            <ul>
+              <li><input type="text" id="text" name="text" maxlength="30" placeholder="Text input" required></li>
+              <li><input type="Email" id="Email" name="email" placeholder="scdk@yahoo.com" required></li>
+            </ul>
+            <textarea name="user_message" id="message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
+            <button class="ctn-btn">See Project</button>
+          </form>
+        </div>
+        <h2 class="align-text">Let's chat!</h2>
+      <p class="align-text">If you have an application you are interested in developing, a feature that you<br> need
+        built or a project that needs coding. I’d love to help with it</p>
+        <div class="ctr-form">
+          <form class="align-text" action="https://formspree.io/f/form_id" method="post">
+            <ul>
+              <li><input type="text" id="name" name="text_input" maxlength="30" placeholder="Full-name" required></li>
+              <li><input type="text" id="user_email" name="email" placeholder="Email address" required></li>
+            </ul>
+            <textarea name="message" id="user_message" cols="30" rows="10" maxlength="500" required>Write your message here</textarea>
+            <button class="ctn-btn-2">See Project</button>
+          </form>
+        </div>
+    </div>
+    <footer class="dkt-fter">
+      <hr class="dkt">
+      <div class="btm-icon-2">
+        <div class="social-2">
+          <i class="fab fa-github fa-2x"></i>
+          <i class="fab fa-facebook fa-2x"></i>
+          <i class="fab fa-instagram fa-2x"></i>
+          <i class="fab fa-twitter fa-2x"></i>
+        </div>
+      <p class="align-text">emailhere@email.com</p>
+    </div>
+    </footer>
```

Comment:
- [ ] Please fix the positioning of your form to be in the center like the Figma design.
- [ ] Please add the correct margin-bottom under social links and email to match the Figma design and kindly fix the width and height of the social media images.
- [ ] Kindly add the horizontal line above the social links and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/161055816-3572a639-207f-44f7-ba8f-4bd1ae850cef.png)  | ![image](https://user-images.githubusercontent.com/68796232/161055861-4f30ae6f-af87-4a6b-8233-d6815b167ade.png)  |